selfdrive/test/test_car_models.py

```diff
@@ -449,6 +457,9 @@ def get_route_log(route_name):
   TOYOTA.HIGHLANDER,
   TOYOTA.HIGHLANDERH,
   TOYOTA.HIGHLANDERH_TSS2,
+  SUBARU.FORESTER_PREGLOBAL,
+  SUBARU.OUTBACK_PREGLOBAL,
+  SUBARU.OUTBACK_PREGLOBAL_2018,
```

Comment:
no need to add these here for dashcam only cars

Reply:
ok

Reply:
SUBARU.OUTBACK_PREGLOBAL_2018 was needed since it does not have a route